cmd/lambda/leases/main.go

```diff
@@ -127,6 +147,53 @@ func init() {
 	muxLambda = gorillamux.New(r)
 }
 
+// initConfig configures package-level variables
+// loaded from env vars.
+func initConfig() {
+	cfgBldr := &config.ConfigurationBuilder{}
+	Settings = &leaseControllerConfiguration{}
+	if err := cfgBldr.Unmarshal(Settings); err != nil {
+		log.Fatalf("Could not load configuration: %s", err.Error())
+	}
+
+	// load up the values into the various settings...
+	err := cfgBldr.WithEnv("AWS_CURRENT_REGION", "AWS_CURRENT_REGION", "us-east-1").Build()
+	if err != nil {
+		log.Printf("Error: %+v", err)
+	}
+	svcBldr := &config.ServiceBuilder{Config: cfgBldr}
+
+	_, err = svcBldr.
+		// AWS services...
+		WithDynamoDB().
+		WithSTS().
+		WithS3().
+		WithSNS().
+		WithSQS().
+		// DCE services...
+		WithStorageService().
+		WithLeaseDataService().
+		WithLeaseManager().
+		WithLeaseService().
+		Build()
+	if err != nil {
+		panic(err)
+	}
+
+	Services = svcBldr
+
+	//resetQueueURL = Config.GetEnvVar("RESET_SQS_URL", "DefaultResetSQSUrl")
+	leaseAddedTopicARN = Config.GetEnvVar("LEASE_ADDED_TOPIC", "DCEDefaultProvisionTopic")
```

Comment:
Didn't we already retrieve this [when we unmarshalled the env vars into the Settings struct](https://github.com/Optum/dce/pull/240/files#diff-6433aa3e05979c963e2ea37c8e383bc0R154) above?

Reply:
For now we are supporting both old and new ways of interacting with db layer.